core/lib/spree/core/controller_helpers/current_host.rb

```diff
@@ -7,7 +7,10 @@ module CurrentHost
         extend ActiveSupport::Concern
 
         included do
-          Spree::RailsCompatibility.active_storage_set_current(self)
+          Spree::Deprecation.warn <<~MSG
```

Comment:
Isn't still breaking this way? We should also add the ActiveStorage module so that people have time to see the deprecation working with the working app.

Reply:
Oh, you're right 🤦 Fixed